Add contact form logic

oltionchampari opened the pull request:
Closes Caleydo/tdp_bi_bioinfodb#1222

### Summary 
* Added the functionality to open the local client when the contact form is submitted.

### Screenshot
![grafik](https://user-images.githubusercontent.com/51322092/117019904-2a819600-acf6-11eb-8005-b5d0994d4cae.gif)

oltionchampari commented on src/pages/components/OrdinoContactForm.tsx:
```diff
@@ -1,47 +1,89 @@
                     <Form.Group >
                         <Form.Label>Email address</Form.Label>
-                        <Form.Control type="email" placeholder="name@example.com" />
+                        <Form.Control type="email" value={email} name="email" placeholder="name@example.com" onChange={handleInputChange} />
```
I think it's redundant having the user enter the email and name since we are opening the local client anyways.
@thinkh If you agree then I can remove this.

thinkh replied:
Yes, I agree. We can remove both fields `name` and `email` as they are covered by the mail client.

thinkh commented on src/pages/components/OrdinoContactForm.tsx:
```diff
@@ -1,47 +1,81 @@
-                            <option>I want to report a bug</option>
-                            <option>Placeholder</option>
+                        <Form.Control onChange={handleInputChange} value={subject} name="subject" as="select">
+                            <option value="I have a question">I have a question</option>
+                            <option value="I want to report a bug">I want to report a bug</option>
```
Please add a general feedback as first option:

```suggestion
                            <option value="I have some general feedback">I have some general feedback</option>
                            <option value="I have a question">I have a question</option>
                            <option value="I want to report a bug">I want to report a bug</option>
```

thinkh reviewed (changes requested):
Thanks for your implementation! Unfortunately, it does not work reliably. When I enter some text into the message textarea and click _send message_ the mail client shows me an outdated state. That is especially appearent when I copy and paste some text into the textarea.

![ordino-contact-form](https://user-images.githubusercontent.com/5851088/117122249-fd84c000-ad95-11eb-8f2f-14b215b5aecc.gif)

My assumption would be that the React state oe renderering of the submit link afterwards is too slow or not work correctly? Maybe a regular form with an `onsubmit` function that then collects the form data is the better option?

oltionchampari commented:
Thanks for your feedback!
As per your recmendation i refactored the code to use a form + ref implemetation. This simplifies the code a lot and fixes the two bugs mentioned above.
Removed the name, email inputs and added a rest to the form.

thinkh reviewed (approved):
Looks good and works now as expected. 👍
